README.md

```diff
@@ -1,6 +1,8 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+## Making a change here to demonstrate how Sema works. 
+
```

Comment:
Radomir, this code is awesome and elegant
As a part of the “fail fast” approach, public methods should always validate input parameters before executing any logic. This way, you avoid getting an error later, where the cause is harder to find.

📄 [Toptal](https://www.toptal.com/java/tips-and-practices)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Elegant

Reply:
we always say this over and over